src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -333,6 +347,37 @@ suggestSignature isQuickFix Diagnostic{_range=_range@Range{..},..}
               | otherwise                  = nm <> ty
 suggestSignature _ _ = []
 
+suggestNewImport :: ExternalPackageState -> ParsedModule -> Diagnostic -> [(T.Text, [TextEdit])]
+suggestNewImport eps ParsedModule {pm_parsed_source = L _ HsModule {..}} Diagnostic{_message}
+  | Just [name, _typ] <- matchRegex (unifySpaces _message) "Variable not in scope: ([^ ]+) :: ([^*•]+)"
```

Comment:
GHC's error message does not always contain the type. E.g. in
```
foo = fromJust
```
The error reads
```
Variable not in scope: fromJust
```
This suggestion doesn't use the type anyway. Could you change the regex to make the type annotation optional?